packages/theme/src/borders/base.ts

```diff
@@ -3,21 +3,18 @@ import { colors } from "../colors";
 
 const baseBorders = {
   none: "none",
-  "1SolidGray": `1px solid ${colors.gray}`,
-  "1SolidGrayLight": `1px solid ${colors.grayLight}`,
-  "2SolidGrayLight": `2px solid ${colors.grayLight}`,
-  "3SolidBlue": `3px solid ${colors.blue}`,
-  "1SolidGreen": `1px solid ${colors.green}`,
-  "1SolidMagenta": `1px solid ${colors.magenta}`,
-  "3SolidMagenta": `3px solid ${colors.magenta}`,
-  "1SolidNavy": `1px solid ${colors.navy}`,
+  "1SolidCritical": `1px solid ${colors.border.critical}`,
+  "1SolidDefault": `1px solid ${colors.border.default}`,
+  "1SolidSubdued": `1px solid ${colors.border.subdued}`,
+  "1SolidSuccess": `1px solid ${colors.border.success}`,
```

Comment:
While we are in this realm... an option could be to drop the 1Solid from the variable name. If we needed to override the size or style we could do that with additional props, but let it be 1px solid by default, or whatever design wants the default size and style to be
```
<Box border="critical" borderSize="2px" borderStyle="dashed" />

Reply:
Lol... if I had just scrolled a little more it looks like that's kinda what you are doing with the base aliases

Reply:
I like your idea much better!